### DISCUSSION TITLE
First param of Op.Function

### DISCUSSION THREAD

**Discussion by cristianvogel** [cables-gl/cables_docs#511] [! Welcome - readme first!]
... doesn't seem to go through to the Parameter 1 output.

<img width="447" alt="Screenshot 2023-03-30 at 22 30 57" src="https://user-images.githubusercontent.com/12829050/228957456-96ec5a36-1cdb-4511-967e-c9710f4cf8f8.png">

**Comment by timpulver** [comment]
I posted a similar bug report.
The problem is fixed now, see:
https://github.com/cables-gl/cables_docs/issues/594
